### DISCUSSION TITLE
now to play to bluetooth?

### DISCUSSION THREAD

**Discussion by skiphansen** [rern/rAudio#466] [General]
I've just discovered rAudio-1 and installed it on a new Raspberry PI 4, so far I'm **very** impressed!

However, I can't figure out how to play audio to my tablet.  If I connect Bluetooth and play something on the tablet the audio come out the Pi, great!  How do I play a song on the rAudio device and have it come out on the tablet?

Thanks for the code!

Skip

**Comment by skiphansen** [comment]
Actually what I really want to do is pair rAudio with a Beats Pill Bluetooth speaker.  Is this possible?

**Comment by skiphansen** [comment]
Never mind ... I got it to work!

Here's how to do it for the next person:

ssh into raudio and follow https://gist.github.com/actuino/9548329d1bba6663a63886067af5e4cb#pair-and-connect 
Then play away.

NB: exit bluetoothctl when you are done or the audio will be **very** broken

**Reply by rern** [reply]
Connect to a Bluetooth speaker/headphone: 
- Settings > Networks > Bluetooth
- Once connected, power on/off will reconnect/disconnect automatically.
